app/jobs/update_clinical_trials.rb

```diff
@@ -0,0 +1,28 @@
+class UpdateClinicalTrials < ActiveJob::Base
+  def perform(recurring = true)
+    begin
+      Source.where(source_type: 'PubMed').each do |source|
```

Comment:
I wonder if these ever change after they've been added or if it would be safe to just grab Pubmed sources that don't have trials associated with them.

Reply:
Not so much change but I could imagine additional ones being added at a later date.

Reply:
Yeah good point, I'd just leave it as is.